.gitignore

```diff
@@ -1,17 +1,3 @@
-# Windows Files
-Desktop.ini
```

Comment:
Honestly, it's really just a pedantic preference of mine to have the smallest `.gitignore` file possible, sans comments and sans stuff that isn't an actual issue :blush:

I don't care too much about it if you prefer a more verbose `.gitignore`?